Documentation/Makefile

```diff
@@ -48,7 +48,7 @@ epub latex html: builder-image copy-api
 
 html-netlify: copy-api
 	@$(ECHO_GEN)_build/$@
-	$(QUIET) ./check-build.sh html $(SPHINX_OPTS)
+	$(QUIET) SKIP_LINT=1 ./check-build.sh html $(SPHINX_OPTS)
```

Comment:
Is this change intended?

Reply:
yes i updated the commit message. this basically skips spell-checking for netlify build. it's already checked in another status check, and we had inconsistent results because netlify image has a different dictionary.